packages/web-components/src/card/card.styles.ts

```diff
@@ -0,0 +1,193 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
```

Comment:
Please use exact imports or bundlers like swc/esbuild will bring over all the components when you import `fast-foundation`, so in this case use `'@microsoft/fast-foundation/utilities.js'`